internal/services/securitycenter/security_center_server_vulnerability_assessment_arc_virtual_machine_resource.go

```diff
@@ -0,0 +1,170 @@
+package securitycenter
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/security/mgmt/v3.0/security"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/tf"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	computeParse "github.com/hashicorp/terraform-provider-azurerm/internal/services/compute/parse"
+	computeValidate "github.com/hashicorp/terraform-provider-azurerm/internal/services/compute/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/securitycenter/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/timeouts"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+const (
+	hybridProvider = "Microsoft.HybridCompute"
+	hybridType     = "machines"
+)
+
+func resourceServerVulnerabilityAssessmentArcVirtualMachine() *pluginsdk.Resource {
```

Comment:
since we can't provision these right now we'll have to remove this for the moment

I've opened https://github.com/hashicorp/terraform-provider-azurerm/issues/15990 to track support for Hybrid Machines, which'll allow us to add acceptance tests for this